Service Portal/Search Sources/DataFetchScript.js

```diff
@@ -0,0 +1,51 @@
+(function(query) {
+    var results = [];
+    var getuserdelegateof = new GlideRecord('sys_user_delegate');
+    getuserdelegateof.addQuery('delegate', gs.getUserID());
+	getuserdelegateof.addEncodedQuery("endsONToday@javascript:gs.beginningOfToday()@javascript:gs.endOfToday()");
```

Comment:
By using 'ON' this code could technically run outside delegation window

I'd suggest something like
`starts<=javascript:gs.endOfCurrentMinute()^ends>=javascript:gs.beginningOfCurrentMinute()`